### ISSUE TITLE
It is not possible to hide the FlotingActionsMenu

### ISSUE THREAD

**Issue by ShreyashPromact** [futuresimple/android-floating-action-button#168]
One of the case in my app, i want to hide the Menu. That's why i have set it to fabMenu.setVisibility(View.GONE); // where fabMenu is my FloatingActionsMenu. But it is not hiding. Please let me know how it can be resolved.

**Comment by moradgholi** [comment]
same here, i'm using a SlidingUpPanel layout and i need to hide the fabMenu and open my sliding panel.
The problem is when i even change the visibility of fabMenu to GONE, it will appear when i tap to that location in my SlidingUpPanel.

**Comment by Poeschl** [comment]
Try to get the "real" button of the menu. It has the id `R.id.fab_expand_menu_button`. If you hide that one, it works for me.

**Comment by Ph1b** [comment]
That was really helpful. I created a circualr reveal on the fam and got a wrong position.
The README should contain some examples and explanations, including that the id of the fam is not the id specified in the layout.

**Comment by Ph1b** [comment]
And it seems to be a good idea, to have the

OnFloatingActionsMenuUpdateListener have a method having the Floating Action Button, representing the FAM as a parameter, like:

    public void onMenuExpanded(FloatingActionButton fabRepresentingFam) {
        ...
    }

**Comment by ShreyashPromact** [comment]
@Poeschl  I want to hide the menu not button in the menu. Do you have any idea for same?

**Comment by Poeschl** [comment]
@ShreyashPromact Which button do you mean exactly? If you mean the button to close the menu again, it is the same button as to open it.

**Comment by ShreyashPromact** [comment]
@Poeschl yes i can able to hide that. But the thing is, when i click on that invisible are, it shows me the menu expanded button as shown in the repository readme file.

**Comment by Poeschl** [comment]
Ah ok, but for that I have no idea ;)

**Comment by ShreyashPromact** [comment]
@Poeschl Thanks for quick reply. If you can able to hide the menu, it will still remain clickable in that hidden location. Just try it. Let me know if you have any solution to that.

Sincerely,
Shreyash
